src/lib.rs

```diff
@@ -663,8 +675,8 @@ impl LevelFilter {
     ///
     /// Returns `None` if `self` is `LevelFilter::Off`.
     #[inline]
-    pub fn to_level(&self) -> Option<Level> {
-        Level::from_usize(*self as usize)
+    pub fn to_level(self) -> Option<Level> {
```

Comment:
Still a breaking change.

Reply:
Sorry, I've forgotten about this one, now I've removed it and also the `dyn` modifiers as well.